packages/adapter-netlify/README.md

```diff
@@ -36,6 +36,15 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 
 It's recommended that you add the `build` and `functions` folders (or whichever other folders you specify) to your `.gitignore`.
 
-## Extra Resources
+## Troubleshooting
```

Comment:
"Troubleshooting" to me means something like debugging, which doesn't really seem accurate. Maybe something more like "Customizing Setup"

Reply:
But isn't it debugging?! I'm thinking of "hey, my Netlify redirect rule doesn't work. Why???" ...I reworded it a little to give a hint on what to do if something isn't working.

Reply:
I guess any piece of info could be framed as either documentation or debugging, but I'd prefer to frame it as documentation. It's better that you read this stuff up front rather than waiting until something's broken and then have to debug

Reply:
Thanks for the explanation. I removed the changes as suggest below.